openmetadata-ui/src/main/resources/ui/src/components/Domain/DomainTabs/DocumentationTab/DocumentationTab.component.tsx

```diff
@@ -27,69 +28,75 @@ import {
+import React, { ReactNode, useCallback, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
 import { getEntityName } from 'utils/EntityUtils';
+import {
+  getChangedEntityNewValue,
+  getChangedEntityOldValue,
+  getDiffByFieldName,
+  getDiffValue,
+  getEntityVersionByField,
+} from 'utils/EntityVersionUtils';
 import { checkPermission } from 'utils/PermissionsUtils';
 import '../../domain.less';
 import { DocumentationTabProps } from './DocumentationTab.interface';
 
-const DocumentationTab = ({ domain, onUpdate }: DocumentationTabProps) => {
+const DocumentationTab = ({
+  domain,
+  onUpdate,
+  isVersionsView,
+}: DocumentationTabProps) => {
   const { t } = useTranslation();
   const { permissions } = usePermissionProvider();
   const [isDescriptionEditable, setIsDescriptionEditable] =
     useState<boolean>(false);
   const [editDomainType, setEditDomainType] = useState(false);
 
-  const editDescriptionPermission = useMemo(
-    () =>
-      checkPermission(
+  const { editDescriptionPermission, editOwnerPermission, editAllPermission } =
+    useMemo(() => {
+      if (isVersionsView) {
+        return {
+          editDescriptionPermission: false,
+          editOwnerPermission: false,
+          editAllPermission: false,
+        };
+      }
+
+      const editDescription = checkPermission(
         Operation.EditDescription,
         ResourceEntity.DOMAIN,
         permissions
-      ) ||
-      checkPermission(Operation.EditAll, ResourceEntity.DOMAIN, permissions),
-    [permissions]
-  );
+      );
 
-  const editOwnerPermission = useMemo(
-    () =>
-      checkPermission(
+      const editOwner = checkPermission(
         Operation.EditOwner,
         ResourceEntity.DOMAIN,
         permissions
-      ) ||
-      checkPermission(Operation.EditAll, ResourceEntity.DOMAIN, permissions),
-    [permissions]
-  );
+      );
 
-  const editAllPermission = useMemo(
-    () =>
-      checkPermission(Operation.EditAll, ResourceEntity.DOMAIN, permissions),
-    [permissions]
-  );
+      const editAll = checkPermission(
+        Operation.EditAll,
+        ResourceEntity.DOMAIN,
+        permissions
+      );
 
-  const onDescriptionUpdate = async (updatedHTML: string) => {
-    if (domain.description !== updatedHTML) {
-      const updatedTableDetails = {
-        ...domain,
-        description: updatedHTML,
+      return {
+        editDescriptionPermission: editDescription || editAll,
+        editOwnerPermission: editOwner || editAll,
+        editAllPermission: editAll,
       };
-      onUpdate(updatedTableDetails);
-      setIsDescriptionEditable(false);
-    } else {
-      setIsDescriptionEditable(false);
-    }
-  };
+    }, [permissions, isVersionsView]);
 
   const getOwner = useCallback(
-    (owner?: EntityReference) => {
+    (ownerDisplayName: string | ReactNode, owner?: EntityReference) => {
```

Comment:
```suggestion
    (ownerDisplayName: ReactNode, owner?: EntityReference) => {
```

ReactNode contains string type